### PR TITLE
show load more button #419

### LINKED ISSUE #419: show load more button
child of #387

### PR DESCRIPTION
**Pull request by artem-rozhkov-dev-optima**
closes #419

### CODE REVIEW

File: src/_designs/components/loadMoreButton/index.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    position: relative;
+    min-height: 60px;
+    &::after {
+      content: "";
+      display:block;
+      position:absolute;
+      width: 100%;
+      left:0;
+      top: 50%;
+      border-bottom: 2px solid $color-neutral-positive-2;
```
Comment (emilsomm) [inline src/_designs/components/loadMoreButton/index.vue]: can you add a transform on this, `transform: translateY(-50%);` to truly center it on the button

File: src/_designs/components/loadMoreButton/index.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    .ae-button {
+      max-width: 310px;
```
Comment (emilsomm) [inline src/_designs/components/loadMoreButton/index.vue]: also avoid px based values as they are completly static

File: src/_designs/components/loadMoreButton/index.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      min-width: 260px;
+      position:absolute;
```
Comment (emilsomm) [inline src/_designs/components/loadMoreButton/index.vue]: this doesnt have to be positioned. As we can use a negative z-index on the sudo element to go under the button, right?

File: src/_designs/components/loadMoreButton/index.vue (hunk 1 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/loadMoreButton/index.vue]: updated

File: src/_designs/components/loadMoreButton/index.vue (hunk 2 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/loadMoreButton/index.vue]: updated

File: src/_designs/components/loadMoreButton/index.vue (hunk 3 above)
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/loadMoreButton/index.vue]: updated

**Comment by emilsomm** [comment]
LGTM 🥇

**Pull request merged**